validator.js

```diff
@@ -99,7 +99,8 @@
       'nn-NO': /^(\+?47)?[49]\d{7}$/,
       'vi-VN': /^(0|\+?84)?((1(2([0-9])|6([2-9])|88|99))|(9((?!5)[0-9])))([0-9]{7})$/,
       'en-NZ': /^(\+?64|0)2\d{7,9}$/,
-      'en-IN': /^(\+?91|0)?[789]\d{9}$/
+      'en-IN': /^(\+?91|0)?[789]\d{9}$/,
+      'de-DE': /^([\+][0-9]{1,3}[ \.\-])?([\(]{1}[0-9]{1,6}[\)])?([0-9 \.\-\/]{3,20})((x|ext|extension)[ ]?[0-9]{1,4})?$/
```

Comment:
Also, this is too verbose. The other mobile phone validators expect that you've stripped non-numeric characters.